src/evox/algorithms/de_variants/ode.py

```diff
@@ -0,0 +1,138 @@
+from typing import Literal
+
+import torch
+
+from ...core import Algorithm, Mutable, Parameter, jit_class
+from ...utils import clamp
+
+
+@jit_class
+class ODE(Algorithm):
```

Comment:
You also need a docstring for this class, otherwise, you get an empty page in the documentation website.
Any 

```python
@jit_class
class ODE(Algorithm):
    """Tell us about this algorithm, and maybe provide a link to its paper here"""
```